src/include/net_connect.h

```diff
@@ -96,30 +96,28 @@ typedef unsigned long pbs_net_t;        /* for holding host addresses */
  **	Types of Inter Server messages (between Server and Mom).
  */
 #define IS_NULL                         0
-#define IS_CLUSTER_ADDRS                1
-#define IS_UPDATE                       2
-#define IS_RESCUSED                     3
-#define IS_JOBOBIT                      4
-#define IS_BADOBIT                      5
-#define IS_REPLYHELLO                   6
-#define IS_SHUTDOWN                     7
-#define IS_IDLE                         8
-#define IS_ACKOBIT                      9
-#define IS_REGISTERMOM                  10
-#define IS_UPDATE2                      11
-#define IS_DISCARD_JOB                  12
-#define IS_DISCARD_DONE                 13
-#define IS_UPDATE_FROM_HOOK             14 /* request to update vnodes from a hook running on parent mom host */
-#define IS_RESCUSED_FROM_HOOK           15 /* request from child mom for a hook */
-#define IS_HOOK_JOB_ACTION              16 /* request from hook to delete/requeue job */
-#define IS_HOOK_ACTION_ACK              17 /* acknowledge a request of the above 2    */
-#define IS_HOOK_SCHEDULER_RESTART_CYCLE 18 /* hook wish scheduler to recycle */
-#define IS_HOOK_CHECKSUMS               19 /* mom reports about hooks seen */
-#define IS_UPDATE_FROM_HOOK2            20 /* request to update vnodes from a hook running on a parent mom host or an allowed non-parent mom host */
-#define IS_HELLOSVR                     21 /* hello send to server from mom to initiate a hello sequence */
-
-#define IS_CMD          40
-#define IS_CMD_REPLY    41
+#define IS_CMD                          1
```

Comment:
actually enum could be dangerous if you suddenly comment one of the numbers out, the next in the series would have their values changed. I would say keeping the numbers the same by using hash defines and providing explicit values to them makes for more interoperability (not that we support that first class, but we would need to going forward)

Reply:
The scheduler uses enums for things like this.  The thing is it will explicitly set the values.  If you were to delete one symbol, nothing changes.  It more or less is the same thing, except that when you are debugging you see the symbol instead of the number.

Reply:
yeah - that would work, but takes away most of the usefulness of having an enum. Yeah i see the nice thing in debugging

Reply:
After discussion with @subhasisb, right now we are not changing this to enum, but at time of flat-buffer (or in future) will change to enum if needed...